src/source/google_analytics_4/GoogleAnalyticsPlugin.bs

```diff
@@ -59,6 +59,13 @@ class GoogleAnalyticsPlugin extends zapp.analytics.BaseAnalyticsPlugin
     m.setInstance("googleAnalyticsTask", m.task)
   end function
 
+  public override function onVideoAnalyticsEvent(event as zapp.analytics.AnalyticsEvent)
+    if m.task = invalid
```

Comment:
let's add some logging here to do 2 things..
1. log to say " this plugin does not have a google analtyics task, using the shared instance"
2. check if m.task is invalid, and if so, log an error and return (save us going crazy if something weird happens).

Reply:
done